pom.xml

```diff
@@ -616,6 +617,19 @@
             </modules>
         </profile>
 
+        <profile>
+            <id>spark-2.4</id>
```

Comment:
Again, why do we need two different profiles here?

Reply:
I answered in the previous question.  These profiles used for examples. Spark 2.3 and 2.4 aren't compatible and should be separated.